### PR TITLE
Homework4: Pavel Klepcha

### PR DESCRIPTION
**Pull request by klep4a**

### CODE REVIEW

File: src/Homework4/02_goroda.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+Novgorod
+
+Выходные данные
+Ukraine
+Russia
+Russia
+"""
+
+dct_sities = {}
+for i in(range(int(input('количество стран? ')))):
```
Comment (belovmd) [inline src/Homework4/02_goroda.py]: redundant parentheses

File: src/Homework4/02_goroda.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+"""
```
Comment (belovmd) [inline src/Homework4/02_goroda.py:1]: What about Brest in France and Belarus?

File: src/Homework4/05_jazyki.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+"""
+
+langs = {}
+for i in range(int(input('количество школьников? '))):
+    langs[i] = set()
+    for j in range(int(input('количество языков школьника? '))):
+        langs[i].add(input('язык? '))
+al_ = set.intersection(*langs.values())
+one_ = set.union(*langs.values())
+print(len(al_), *list(al_), len(one_), *list(one_), sep='\n')
```
Comment (belovmd) [inline src/Homework4/05_jazyki.py]: it would be nice if we'll add description what is what :)

File: src/Homework4/07_nod_evklid.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+при помощи алгоритма Евклида (мы не знаем функции и рекурсию).
+"""
+
+small, big = 215484565498465686464846546846516841646568166851, 462
+while big and small:
+    if big > small:
+        big %= small
+    else:
+        small %= big
+print(big + small)
```
Comment (belovmd) [inline src/Homework4/07_nod_evklid.py]: it's a cheat. algorithm doesn't expect result in sum :) 
you can use ternary operator.

**Review by belovmd — CHANGES_REQUESTED** [review]
Please take a look at these issues

File: src/Homework4/07_nod_evklid.py (hunk 4 above)
Comment (klep4a) [reply src/Homework4/07_nod_evklid.py]: Euclid would understand :-)

File: src/Homework4/05_jazyki.py (hunk 3 above)
Comment (klep4a) [reply src/Homework4/05_jazyki.py]: added

File: src/Homework4/02_goroda.py (hunk 2 above)
Comment (klep4a) [reply src/Homework4/02_goroda.py:1]: solved

**Review by belovmd — COMMENTED** [review]
9

**Pull request closed without merging**